### ISSUE TITLE
any chance of converting it to v8?

### ISSUE THREAD

**Issue by jacksolutions** [alan196/tms#1]
i tried to install it on odoov8 but no luck, is it converted it to v8? any chance of converting it to v8?

**Comment by alan196** [comment]
yes, it is in https://github.com/odoo-jarsa/transport_management_system branch 8.0
At this point you can install it in V8 but it will not work 100%. It's WIP but we don't have defined when it will be done.
